docs/dev/test-configuration-options.md

```diff
@@ -279,121 +306,248 @@ Specifies the screen resolution to be used during your test session. Default scr
 
-Appium automatically determines the package to launch; you'll only need to use this capability if you want to specify a package different from the default one.
-:::
+Allows you to set a path to an `.ipa`, `.apk`, `.aab` or `.zip` file containing the mobile app you want to test. This could be the location of your app in [App Storage](/mobile-apps/app-storage) (e.g., `storage:filename=myapp.zip`) or the URL to a remote location where your app is located (e.g., `http://myappurl.zip`). The remote location needs to be accessible from the web, Sauce Connect can not access your internal file system where apps are hosted.
 
+:::note
+- If this capability is not provided for a virtual device, the ['browserName'](#browserName) capability needs to be set. If none is set the test will throw an error.
+- This capability can be omitted for virtual devices if the ['browserName'](#browserName) capability is set.
+- If this capability is not provided for a real device session and also the ['browserName'](#browserName) capability is not provided then a real device session will automatically default back to the default browser. This will be Chrome for Android and Safari for iOS
+:::
 
 ```java
-"appium:appPackage": "com.example.android.myApp, com.android.settings"
+MutableCapabilities capabilities = new MutableCapabilities();
+capabilities.setCapability("appium:app", "storage:filename=my_app.zip");
 ```
 
+:::tip Using Storage Id
+If you have an app you have uploaded to [Sauce storage](https://app.saucelabs.com/live/app-testing), you can also set the `app` capability to `"storage:xxxxxxxxx-xxxxxxx-xxx"` and enter the **FILE ID** for your app. This allows you to set which specific version you uploaded, otherwise if you use the file name, it will use the latest one uploaded with the exact same name.
+:::
+
 ---
-### `appActivity`
-<p><small>| STRING | <span className="sauceDBlue">Android Only</span> |</small></p>
 
-Allows you to set the name for the Android activity you want to launch from your package. This capability must be preceded by a dot (e.g., `.MainActivity`).
+### `appium:orientation`
 
-:::tip Automatic Activity Detection
+<p><small>| OPTIONAL | STRING | <span className="sauceDBlue">Virtual and Real Devices</span> |</small></p>
 
-Appium automatically determines the activity to launch; you'll only need to use this desired capability if you want to specify an activity different from the default one.
+Specifies the orientation of the screen during the test. Valid values are `PORTRAIT` and `LANDSCAPE`.
+
+:::important
+This capability is an Appium capability and needs to be pre-fixed with `appium:` so it becomes `appium:orientation`. It can be used for virtual device mobile tests **and** for real device tests. It will only flip the screen. The capability `deviceOrientation` will flip the skin **and** the screen, see [`deviceOrientation`](#deviceorientation) for more information.
```

Comment:
```suggestion
This capability is an Appium capability that needs to be pre-fixed with `appium:` so it becomes `appium:orientation`. It can be used for virtual device mobile tests and real device tests. The`appium:orientation` capability will only flip the screen while the capability `deviceOrientation` will flip the skin and the screen. See [`deviceOrientation`](#deviceorientation) for more information.
```